src/StackAdmin/AzureStack/AzureStack.psd1

```diff
@@ -121,25 +121,10 @@
             # IconUri = ''
 
             # ReleaseNotes of this module
-            ReleaseNotes = '## 2018.12 - Version 1.6.0
-            * Supported with 1811 update
-            * Azs.Compute.Admin Module
-                * Fixed missing Azs prefix for New-DataDiskObject and added alias with warning of future deprecation.
-            * Azs.Update.Admin Module
-                * Added a warning to recommend running Test-AzureStack before Install-AzsUpdate
-            * Azs.Fabric.Admin
-                * New cmdlet (The features are supported by Azure Stack 1811+)
-                    * Get-AzsDrive
-                    * Get-AzsVolume
-                    * Get-AzsStorageSubSystem
-                * Deprecation
-                    * Get-AzsInfrastructureVolume is an alias now to the cmdlet Get-AzsVolume
-            * Azs.InfrastructureInsights.Admin
-                *  Added a new cmdlet Repair-AzsAlert
-            * Azs.Storage.Admin
-                * Bug fix where default quota values are not being used
-            * Azs.Subscriptions.Admin Module
-                * Fixed missing Azs prefix for New-AddonPlanDefinitionObject and added alias with warning of future deprecation.'
+            ReleaseNotes = '## 2019.MM - Version 1.7.0
+            * Supported with 1901 update
+            * Azs.Backup.Admin Module
+                * Breaking change: Backup changes to cert-based encryption mode. Support for symmetric keys is deprecated.
```

Comment:
@daozha it looks like this line is missing the trailing single quote, so this module would fail to be imported properly

Reply:
Thanks. Ps-sign wpuld have caught that.. I have kicked off one now

Reply:
Thanks Cormac and Bala.